ivy/functional/frontends/torch/tensor.py

```diff
@@ -875,3 +875,7 @@ def __and__(self, other):
 
     def bitwise_xor(self, other, *, out=None):
         return torch_frontend.bitwise_xor(self._ivy_array, other)
+
+    @with_unsupported_dtypes({"1.11.0 and below": ("bfloat16",)}, "torch")
+    def exp(self, other, *, out=None):
```

Comment:
exp needs only one argument, it takes `self` only, `other` may be removed

Reply:
removed